app/controllers/candidate_interface/other_qualifications/base_controller.rb

```diff
@@ -60,9 +60,17 @@ def other_qualification_params
 
     def other_qualifcations_update_params
       params.require(:candidate_interface_other_qualification_form).permit(
-        :qualification_type, :subject, :institution_name, :grade, :award_year
-      ).merge!(id: params[:id])
-        .transform_values(&:strip)
+        :subject, :institution_name, :grade, :award_year, :other_uk_qualification_type,
+        :non_uk_qualification_type, :institution_country
+      ).merge!(
+        id: params[:id],
+        qualification_type: params.dig('candidate_interface_other_qualification_form', 'qualification_type' || get_qualification.qualification_type),
```

Comment:
This line doesn't look quite right but I'm not sure:
```suggestion
        qualification_type: params.dig('candidate_interface_other_qualification_form', 'qualification_type') || get_qualification.qualification_type,
```